### ISSUE TITLE
[Feature request] More easily allow sites via right-click integration

### ISSUE THREAD

**Issue by gitressa** [uBlockOrigin/uBlock-issues#2790]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

It would be nice if you could right-click on an image element, and allow it to load by adding local rules for the relevant domains. Maybe something like this?

- Go to a URL like https://www.decathlon.nl/p/fietsponcho-zichtbaar-s-nachts-560-zwart-fluogeel/_/R-p-343167
- Right click on the image element ("Fietsponcho zichtbaar 's nachts 560 zwart/fluogeel")
- Select "Allow this element for current site" **<<< Add this option**

These two site should then get enabled as local rules, and the image allowed to be loaded for the current site:
```
mediadecathlon.com
  contents.mediadecathlon.com

d.akamai.net
  a1813.d.akamai.net
  contents.mediadecathlon.com
```

### A specific URL where the issue occurs.

https://www.decathlon.nl/p/fietsponcho-zichtbaar-s-nachts-560-zwart-fluogeel/_/R-p-343167

### Steps to Reproduce

See the feature request above.

### Expected behavior

See the feature request above.

### Actual behavior

See the feature request above.

### uBO version

1.51.0

### Browser name and version

Firefox

### Operating System and version

Linux

**Comment by gwarser** [comment]
But when elements are blocked, they are not visible. There is nothing to click on.

**Comment by gwarser** [comment]
Duplicate of https://github.com/uBlockOrigin/uBlock-issues/issues/2348 ?

**Comment by gitressa** [comment]
Yes, sorry. It's a duplicate. I'll close here and comment in the other issue.
